src/datepicker/datepicker.js

```diff
@@ -304,7 +314,8 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
   $scope.keys = { 13: 'enter', 32: 'space', 33: 'pageup', 34: 'pagedown', 35: 'end', 36: 'home', 37: 'left', 38: 'up', 39: 'right', 40: 'down' };
 
   var focusElement = function() {
-    self.element[0].focus();
+    var focusTarget = self.element[0].querySelector('[tabindex]:not([tabindex="-1"])');
```

Comment:
this doesn't use jquery either?  :focusable

Reply:
I tried :focusable first without luck.  It's possible something else was broken at the time.  I can give it another try, but I suspect it won't work

Reply:
you'll need do something like self.selement.find(':focusable').first()

Reply:
My guess is that querySelector in this case is not jquery's query and is probably something like the native browsers DOM implementation.